src/test/scala/mesosphere/marathon/core/launchqueue/impl/LaunchQueueActorTest.scala

```diff
@@ -58,6 +58,7 @@ class LaunchQueueActorTest extends AkkaUnitTest with ImplicitSender {
       val app = AppDefinition(PathId("/foo"))
       val instance = TestInstanceBuilder.newBuilder(app.id).addTaskRunning().getInstance()
 
+      @volatile
```

Comment:
Vals cannot be volatile. I wonder if this is deep down in Mockito 🤔

Reply:
Also, volatile guards the reference itself, but not whatever reflection magic mockito does with it, right?

Reply:
Yep. I wonder if Mockito is thread safe.

Reply:
..it's not https://github.com/mockito/mockito/wiki/FAQ#is-mockito-thread-safe.